pkg/controllers/nodeclaim/consistency/controller.go

```diff
@@ -70,7 +70,6 @@ func NewController(clk clock.Clock, kubeClient client.Client, cloudProvider clou
 		recorder:      recorder,
 		lastScanned:   cache.New(scanPeriod, 1*time.Minute),
 		checks: []Check{
-			NewTermination(clk, kubeClient),
```

Comment:
Noting that we are going to add this back in as part of our observability PR on draining (#1837)